Formula/dprint.rb

```diff
@@ -0,0 +1,50 @@
+class Dprint < Formula
+  desc "Pluggable and configurable code formatting platform written in Rust"
+  homepage "https://dprint.dev/"
+  url "https://github.com/dprint/dprint/archive/0.10.0.tar.gz"
+  sha256 "1a17ea8feb9a6b20b1c265f97fb113c329116325da442e65a45b3beba675bc1a"
+  license "MIT"
```

Comment:
need to clarify the license :(

Reply:
Since the author's willing to switch to a more friendly license, maybe this can wait till that happens?

Reply:
it happened, I will remake a new PR following up on this.